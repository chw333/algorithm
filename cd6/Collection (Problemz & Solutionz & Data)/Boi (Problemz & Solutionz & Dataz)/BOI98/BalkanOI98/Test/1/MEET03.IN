148 7 97
3 2 2
8 1 5
10 9 1
14 5 2
21 13 3
21 20 6
22 7 5
24 4 4
26 10 2
33 15 1
35 8 4
37 22 9
39 3 3
41 29 3
43 2 8
47 6 9
48 7 6
50 25 3
51 20 7
52 27 8
53 3 7
54 12 5
55 27 2
55 52 9
56 2 9
56 13 1
56 16 5
56 35 5
57 47 7
58 4 5
58 38 5
60 35 7
61 9 10
62 22 5
63 34 8
64 53 1
66 59 5
72 42 8
72 71 8
73 62 9
74 49 1
76 58 5
77 17 8
77 26 1
77 43 4
78 40 5
79 7 2
79 12 6
79 36 3
79 39 3
80 6 2
81 39 8
81 78 10
83 20 8
84 7 3
84 18 5
84 50 7
84 77 4
85 15 8
85 67 4
86 85 5
87 78 2
88 60 9
88 65 8
88 69 6
89 12 4
89 61 5
90 5 9
90 76 4
91 81 2
92 16 2
92 45 5
93 90 7
95 45 1
96 11 7
97 25 2
97 79 3
98 15 4
98 50 3
99 7 7
99 49 3
99 97 6
100 19 4
103 26 8
104 14 9
105 18 10
106 36 4
106 59 4
107 27 6
107 52 10
107 76 1
108 63 6
109 7 4
109 23 9
109 31 6
109 41 10
110 8 8
110 75 6
111 13 4
111 32 2
112 63 8
113 71 2
114 110 8
115 28 7
115 58 9
115 75 5
116 46 9
116 49 2
116 68 3
117 70 8
118 59 9
119 42 9
119 50 9
121 7 4
121 40 4
121 97 10
122 33 8
123 43 10
124 94 2
125 30 8
125 82 9
125 95 6
125 124 2
126 77 5
126 116 3
127 90 8
127 100 6
128 44 7
128 71 3
128 101 9
129 39 6
129 70 2
130 65 3
130 96 4
130 102 3
132 104 3
132 120 6
133 132 3
134 127 3
135 112 10
136 6 9
136 42 8
137 31 3
138 73 10
139 97 3
139 131 2
139 135 2
140 139 8
