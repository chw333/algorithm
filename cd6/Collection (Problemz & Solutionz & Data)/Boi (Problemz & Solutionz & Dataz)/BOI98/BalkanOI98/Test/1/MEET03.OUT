7 84 7 121
97 121
